containersQa/testlib.bash

```diff
@@ -342,7 +342,7 @@ function mavenCreateAndRun() {
     # collections dont like multiline
     runOnBaseDirBash "`sclEnable` mvn -B archetype:generate -DgroupId=org.test.rhimg  -DartifactId=rhimg -DarchetypeArtifactId=maven-archetype-quickstart -DarchetypeVersion=1.4 -DinteractiveMode=false && cd rhimg && `sclEnable` mvn -B clean install && `sclEnable` java -cp  target/rhimg-1.0-SNAPSHOT.jar org.test.rhimg.App"
   else
-    runOnBaseDirBash "mvn -B archetype:generate -DgroupId=org.test.rhimg  -DartifactId=rhimg -DarchetypeArtifactId=maven-archetype-quickstart -DarchetypeVersion=1.4 -DinteractiveMode=false &&  cd rhimg && mvn -B clean install && java -cp  target/rhimg-1.0-SNAPSHOT.jar org.test.rhimg.App"
+    runOnBaseDirBash "mvn -B archetype:generate -DgroupId=org.test.rhimg  -DartifactId=rhimg -DarchetypeArtifactId=maven-archetype-quickstart -DarchetypeVersion=1.4 -DinteractiveMode=false &&  cd rhimg && sed -i 's;<maven.compiler.source>1.7</maven.compiler.source>;<maven.compiler.source>1.8</maven.compiler.source>;g' pom.xml && sed -i 's;<maven.compiler.target>1.7</maven.compiler.target>;<maven.compiler.target>1.8</maven.compiler.target>;g' pom.xml && mvn -B clean install && java -cp  target/rhimg-1.0-SNAPSHOT.jar org.test.rhimg.App"
```

Comment:
I htink the sed is not necessary. the only think to update is -DarchetypeVersion to something newer.

Reply:
I spent two days looking the newest is 1.4 that defaults to jdk 1.7. It has not been updated since around 2019